### PR TITLE
Packaging

### PR DESCRIPTION
**Pull request by YoniItzhak**

### CODE REVIEW

File: jenkins/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,148 @@
+def secrets = [
+  [path: 'secret/jenkins/github', engineVersion: 2, secretValues: [
+    [envVar: 'PRIVATE_REPO_GITHUB_TOKEN', vaultKey: 'private_token']]],
+]
+
+def configuration = [vaultUrl: "${env.VAULT_URL}",  vaultCredentialId: 'vault-app-role', engineVersion: 2]
```
Comment (warolv) [inline jenkins/Jenkinsfile:6]: I don't see you using vault?

File: jenkins/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,148 @@
+def secrets = [
```
Comment (warolv) [inline jenkins/Jenkinsfile:1]: I don't see you using vault?

File: jenkins/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+    kubernetes {
+      label 'cluster-manager'
+      defaultContainer 'jnlp'
+      yamlFile 'jenkins/build-pod.yaml'
+    }
+  }
+
+  options {
+    checkoutToSubdirectory('cloudify-cluster-manager')
+    buildDiscarder(logRotator(numToKeepStr:'30'))
```
Comment (warolv) [inline jenkins/Jenkinsfile:35]: can you put 10 instead? trying to clean unneeded logs as much as possible

File: jenkins/Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+    timeout(time: 60, unit: 'MINUTES')
+    timestamps()
+  }
+
+  environment {
+    BRANCH = "${env.BRANCH_NAME}"
+    PROJECT = "cloudify-cluster-manager"
+    VERSION=getClusterManagerVersion("${env.BRANCH_NAME}").trim()
+    PRERELEASE = getClusterManagerPreRelease("${env.BRANCH_NAME}").trim()
+    S3_BUILD_PATH = "${env.PROJECT}/${env.VERSION}/${env.PRERELEASE}-build/${env.BRANCH}"
```
Comment (warolv) [inline jenkins/Jenkinsfile:45]: add build number and branch please, to maintain a history, like:    
S3_BUILD_PATH = "${params.VERSION}/${params.PRERELEASE}-build/premium-builder/${params.BRANCH}/${params.BUILD_NUMBER}"

**Review by warolv — CHANGES_REQUESTED** [review]

File: jenkins/Jenkinsfile (hunk 1 above)
Comment (YoniItzhak) [reply jenkins/Jenkinsfile:6]: Correct, I assumed it's needed for connection with GitHub. Removed.

File: jenkins/Jenkinsfile (hunk 2 above)
Comment (YoniItzhak) [reply jenkins/Jenkinsfile:1]: Same as the previous comment.

File: jenkins/Jenkinsfile (hunk 3 above)
Comment (YoniItzhak) [reply jenkins/Jenkinsfile:35]: Sure, changed.

File: jenkins/Jenkinsfile (hunk 4 above)
Comment (YoniItzhak) [reply jenkins/Jenkinsfile:45]: Thanks for the comment, I'll add the build number. This package is supposed to be separated from the releases of Cloudify. Therefore, I think the path should be `s3://cloudify-release-eu/cloudify/cloudify-cluster-manager/${params.VERSION}/${params.PRERELEASE}-build/${params.BRANCH}/${params.BUILD_NUMBER}. This is also the reason why I created `getClusterManagerVersion` and `getClusterManagerPreRelease` functions.
Let me know what you think.

Comment (warolv) [reply jenkins/Jenkinsfile]: sounds good!

**Review by warolv — APPROVED** [review]
Good job!

**Pull request merged**